gitkraken-desktop/gitkraken-organization.md

```diff
@@ -40,6 +40,19 @@ Only members that have a [role](/gitkraken-desktop/gitkraken-organization/#roles
 
 You can also add licenses in bulk to your organization separately from adding users by going to the [Subscription](/gitkraken-desktop/gitkraken-subscriptions/) section in your organization.
 
+
+### Inviting Users to Your Organization
+
+As a User, you have the ability to invite other users to the organization for an Owner, Admin, or Billing Contact to approve and/or purchase a license for. 
+
+To invite a user, navigate to gitkraken.dev > Users > Invite Users. Then provide the email(s) of the desired user(s). 
+
+<img src='/wp-content/uploads/gkdev-invite-user-as-user.png' srcset='/wp-content/uploads/gkdev-invite-user-as-user@2x.png' class='img-bordered img-responsive center'>

```

Comment:
This can be done in GKD - we should explain where to access organization/users and include screenshots from gkd. I am going to merge this but we should make an update to this section.